_editorial_policies/03_reviewer_information.md

```diff
@@ -7,121 +7,46 @@ excerpt: Information for reviewers
-However, we do also have specific [review criteria](https://livecomsjournal.github.io/authors/policies/#review-criteria) and we ask reviewers to make sure they are aware of these, including [category-specific review criteria](https://livecomsjournal.github.io/authors/policies/#types-of-articles) relevant to the particular category of article.
-Additionally, consistent with its pedagogical mission, LiveCoMS requires a student review of each manuscript.
-We also have a Conflict of Interest policy and reviewers should make sure they are familiar with this and decline to review if they have a conflict, or if they are uncertain, to disclose any potential conflicts to the editor managing the review process.
-
-One potentially unique aspect of the LiveCoMS review process is the opportunity to engage with authors publicly rather than anonymously if you so desire.
-Specifically, if you (as a reviewer) are interested in participating public discussion of the work and helping to improve it, you are free to comment on the issue tracker connected to the GitHub repository associated with the article, which can be a way to work together with the community to help authors improve their article.
-If you choose to do this, you are welcome to cross-reference GitHub comments in your review for LiveCoMS.
-
-
-Because we are attempting to make LiveCoMS as open as possible, we provide examples below of typical form letters used for correspondence with reviewers.
-
-## Reviewer letters
-
-### Sample review request letter
-
-Dear XYZ,
-
-We invite you to review the article indicated below, which has been submitted to the `YYY` category of the [Living Journal of Computational Molecular Science](http://livecomsjournal.org) (LiveCoMS).
-LiveCoMS is a brand new and rather unique journal, which focuses on promoting best practices in molecular computation via updateable educational articles of various kinds - tutorials, reviews, best-practices “manuals” and “lessons learned” articles.
-The journal is low-cost, non-profit and run by members of our community in a transparent way; in part, it aims to provide academic credit for important work that often goes unrewarded.
-One unique feature of the review process is the requirement for a student review of every article.
-
-To ACCEPT: `link`
-To DECLINE: `link`
-
-Reviews are generally expected within 14 days, although extra time may be allowable upon request.
-
-Please be sure you review and understand our conflict of interest policies before making a decision on whether or not to accept this review request.
-
-If you have any questions, please reply to this email.
-
-Thank you very much.
-`{{SENDER-FULL-NAME}}`, Associate Editor
+Given this preparation, LiveCoMS asks that reviewers respond promptly to review requests and take immediate action on reviewing manuscripts; typically we ask that reviews be returned within 21 days.
```

Comment:
I thought we'd settled on asking for 14 days/discussed this at some length?  I'm OK either way, I just thought this was a decided policy.